homeassistant/components/renault/renault_entities.py

```diff
@@ -61,10 +61,13 @@ def extra_state_attributes(self) -> Mapping[str, Any] | None:
         elif self.entity_description.coordinator == "location":
             last_update = self._get_data_attr("lastUpdateTime")
         if last_update:
-            return {ATTR_LAST_UPDATE: self._convert_to_utc_string(last_update)}
+            return {ATTR_LAST_UPDATE: _convert_to_utc_string(last_update)}
         return None
 
-    def _convert_to_utc_string(self, value: StateType) -> str:
-        """Convert date to UTC iso format."""
-        original_dt = parse_datetime(cast(str, value))
-        return as_utc(original_dt).isoformat()
+
+def _convert_to_utc_string(value: StateType) -> str:
+    """Convert date to UTC iso format."""
+    original_dt = parse_datetime(cast(str, value))
```

Comment:
If we know that the value is a string we should type it correctly in the signature of this function and not cast it here. Adjust the typing as early as possible in the call flow.

Reply:
I've moved it up.

Reply:
I've moved it higher...